app/optionsdialog.ui

```diff
@@ -87,32 +87,39 @@
          <property name="title">
           <string>Font</string>
          </property>
-         <layout class="QHBoxLayout" name="horizontalLayout">
-          <item>
-           <widget class="QLabel" name="familyLabel">
+         <layout class="QGridLayout" name="gridLayout_3">
+          <item row="0" column="1">
+           <widget class="QFontComboBox" name="fontComboBox"/>
+          </item>
+          <item row="0" column="2">
+           <widget class="QLabel" name="sizeLabel">
             <property name="text">
-             <string>&amp;Family:</string>
+             <string>Si&amp;ze:</string>
             </property>
             <property name="buddy">
-             <cstring>fontComboBox</cstring>
+             <cstring>sizeComboBox</cstring>
             </property>
            </widget>
           </item>
-          <item>
-           <widget class="QFontComboBox" name="fontComboBox"/>
+          <item row="0" column="3">
+           <widget class="QComboBox" name="sizeComboBox"/>
           </item>
-          <item>
-           <widget class="QLabel" name="sizeLabel">
+          <item row="0" column="0">
+           <widget class="QLabel" name="familyLabel">
             <property name="text">
-             <string>Si&amp;ze:</string>
+             <string>&amp;Family:</string>
             </property>
             <property name="buddy">
-             <cstring>sizeComboBox</cstring>
+             <cstring>fontComboBox</cstring>
             </property>
            </widget>
           </item>
-          <item>
-           <widget class="QComboBox" name="sizeComboBox"/>
+          <item row="1" column="0">
+           <widget class="QCheckBox" name="sourceSingleSizedCheckBox">
+            <property name="text">
+             <string>Source at Single Size</string>
```

Comment:
Looks better. Thank you! I'm still not sure about the wording. :smile: 
I'm not sure if the user can make the connection between "Source" and the Markdown Source Editor part.
What do you think about "Markdown Source at Single Size"?